### DISCUSSION TITLE
Release on App Store?

### DISCUSSION THREAD

**Discussion by lolomokakus** [LIJI32/SameBoy#612] [General]
Now that Apple has changed the App Store policies to allow for emulator apps, is there any plan to publish Sameboy on there? I’ve looked around a bit and couldn’t find any comment from the dev on this (although I don’t have Twitter, so I don’t know what they’ve said there).

**Comment by LIJI32** [comment]
SameBoy is currently during App Store review, waiting for Apple's approval.
